README.md

```diff
@@ -79,21 +106,26 @@ export QPC_DBMS=SQLite
 ```
 
 ## Initializing the Server
-To initialize the server with Postgres, run the following command:
```

Comment:
Should we add a note on how to initialize the server with Postgres here, as well?